examples/next/pages/ui/components/attribute-management/SimpleProfilePage.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { useUserAttributes } from '@aws-amplify/ui-react-core';
+import { UserAttributeKey } from 'aws-amplify/auth';
+
+export const SimpleProfilePage = () => {
+  const [state, handleAttributes] = useUserAttributes();
+  const [isEditMode, setIsEditMode] = React.useState(false);
+  const formRef = React.useRef<HTMLFormElement>(null);
+  const updateEmailRef = React.useRef<HTMLFormElement>(null);
+  const verifyEmailRef = React.useRef<HTMLFormElement>(null);
```

Comment:
Are we able to achieve functionality with a single `form`?

Reply:
Yes, the one thing I can't figure out with that implementation is how to clear the input field after a deletion occurs, but I'm sure I'm just missing something simple.

Reply:
Can you use `form.reset()`?

Reply:
I added one form back just to allow for the deletion to clear the input field. Will look into moving deletion to a separate page to avoid this altogether.